src/other/utility/utility-en.hbs

```diff
@@ -0,0 +1,173 @@
+---
+{
+	"title": "Uility",
+	"language": "en",
+	"category": "Other",
+	"description": "Utilities page including working examples.",
+	"tag": "utility",
+	"parentdir": "utility",
+	"altLangPrefix": "utility",
+	"dateModified": "2021-11-08"
+}
+---
+<h2>Background</h2>
+<h3><code>bg-cover</code></h3>
+<p>Set background image to fully cover the box size.</p>
+<h4>working example:</h4>
+<div class="provisional bg-cover" data-bgimg="https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg">
+	<div class="container p-0 p-sm-3">
+		<div class="well header-rwd brdr-0 brdr-rds-0 text-white bg-gctheme opct-90">
+			<h1 property="name" id="wb-cont">Canada.ca</h1>
+			<p class="mrgn-tp-md">The official website of the Government of Canada</p>
+		</div>
+	</div>
+</div>
+<div class="provisional gc-featured-link">
+	<div class="container">
+		<div class="row">
+			<div class="col-xs-12">
+				<h2 class="wb-inv">Featured</h2>
+				<p><a class="stretched-link" href="#">Featured link</a></p>
+			</div>
+		</div>
+	</div>
+</div>
+
+<h3><code>bg-center</code></h3>
+<p>Set background image in the center of the container.</p>
+<h4>working example:</h4>
+<div class="provisional bg-center" data-bgimg="https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg">
+	<div class="container p-0 p-sm-3">
+		<div class="well header-rwd brdr-0 brdr-rds-0 text-white bg-gctheme opct-90">
+			<h1 property="name" id="wb-cont">Canada.ca</h1>
+			<p class="mrgn-tp-md">The official website of the Government of Canada</p>
+		</div>
+	</div>
+</div>
+<div class="provisional gc-featured-link">
```

Comment:
Please revisit this working example, "provisional" and "gc-featured-link" is not in scope of wet-boew, only for gcweb